### PR TITLE
New recipe1

### LINKED ISSUE #22: add hommus recipe in the sides folder
not sure about the quantities.

### PR DESCRIPTION
**Pull request by larabaa**
fix #22

### CODE REVIEW

File: sides/hummus [hunk 1]
```diff
@@ -0,0 +1,9 @@
+Ingredients:
+-1 cup chickpeas
```
Comment (remilaumont) [inline sides/hummus:2]:
```suggestion
- 1 cup chickpeas
```

File: sides/hummus [hunk 2]
```diff
@@ -0,0 +1,9 @@
...
+- 2 spoon Tahini
+- 3 garlic
+- 100 ml of lemon 
+- 1 teaspoon of salt
+- 1000 teaspoon of cumin
```
Comment (remilaumont) [inline sides/hummus:7]:
```suggestion
- 2 teaspoon of cumin
```

**Review by remilaumont — CHANGES_REQUESTED** [review]
Thank you for the receipe. It looks great!
I would add some olive oil. What do you think?

**Review by budziszpiotr — APPROVED** [review]
thank you for this nice recipe, sounds delicious

**Review by s194317 — CHANGES_REQUESTED** [review]
I LOVE hummus. Thanks for adding it! Can we add more tahini? Do you mean 3 garlic cloves or 3 whole garlic?

**Review by jakobtkn — APPROVED** [review]
This looks delicious. Thank you @larabaa!

**Comment by larabaa** [comment]
> I LOVE hummus. Thanks for adding it! Can we add more tahini? Do you mean 3 garlic cloves or 3 whole garlic?

3 garlic cloves

**Review by chaozg — APPROVED** [review]
Looks very good to me

**Pull request closed without merging**